CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 2.6.0
+- Moved SimpleCov configuration to the logstash repo 
```

Comment:
```suggestion
- Removed SimpleCov configuration, as it will be moved directly to elastic/logstash [#103](https://github.com/elastic/logstash-devutils/pull/103)
```